### ISSUE TITLE
Yggdrasil IPv6 from EmerCoin resolving 

### ISSUE THREAD

**Issue by olej-tsil-51** [Revertron/Alfis#350]
Alfix build from sources https://github.com/Revertron/Alfis.git, DNS resolving 127.0.0.1:53 work OK:
`olej@R420:~$ nslookup myip.ygg 127.0.0.1
Server:		127.0.0.1
Address:	127.0.0.1#53

Name:	myip.ygg
Address: 200:6223::d35b:1fd8:be0d:2841
`
But for addresses from EmerCoin domains via OpenNIC DNS, such a strange error occurs
`olej@R420:~$ host ygg.linux-ru.lib 127.0.0.1 -p53
;; Got bad packet: FORMERR
110 bytes
a1 0e 81 83 00 01 00 00 00 01 00 00 03 79 67 67          .............ygg
08 6c 69 6e 75 78 2d 72 75 03 6c 69 62 00 00 01          .linux-ru.lib...
00 01 00 00 00 06 00 01 00 00 09 42 00 40 01 61          ...........B.@.a
0c 72 6f 6f 74 2d 73 65 72 76 65 72 73 03 6e 65          .root-servers.ne
74 00 05 6e 73 74 6c 64 0c 76 65 72 69 73 69 67          t..nstld.verisig
6e 2d 67 72 73 03 63 6f 6d 00 78 95 52 7c 00 00          n-grs.com.x.R|..
07 08 00 00 03 84 00 09 3a 80 00 01 51 80                ........:...Q.

olej@R420:~$ host linux-ru.lib 127.0.0.1 -p53
;; Got bad packet: FORMERR
106 bytes
d3 43 81 83 00 01 00 00 00 01 00 00 08 6c 69 6e          .C...........lin
75 78 2d 72 75 03 6c 69 62 00 00 01 00 01 00 00          ux-ru.lib.......
00 06 00 01 00 00 09 0f 00 40 01 61 0c 72 6f 6f          .........@.a.roo
74 2d 73 65 72 76 65 72 73 03 6e 65 74 00 05 6e          t-servers.net..n
73 74 6c 64 0c 76 65 72 69 73 69 67 6e 2d 67 72          stld.verisign-gr
73 03 63 6f 6d 00 78 95 52 7c 00 00 07 08 00 00          s.com.x.R|......
03 84 00 09 3a 80 00 01 51 80                            ....:...Q.
`
Via OpenNIC DNS it's OK:
`olej@R420:~$ host ygg.linux-ru.lib 94.16.114.254
Using domain server:
Name: 94.16.114.254
Address: 94.16.114.254#53
Aliases: 

ygg.linux-ru.lib has IPv6 address 221:58c9:9a6:99be:f3d:c1ac:2b5b:9771
`
OpenNIC DNS is listed in /etc/alfis.conf as :
`forwarders = [94.16.114.254:53, 94.247.43.254:53, 194.36.144.87:53]`

**Comment by olej-tsil-51** [comment]
I assume that EmerCoin 's root domains: .coin, .emc, .lib, .bazar are not recognized as top-level domain names and are not passed to forwarders

**Comment by Revertron** [comment]
No, You just have one forwarder, that doesn't know about those zones, and it is chosen by random.

**Comment by olej-tsil-51** [comment]
May be ... But:
> root@R420:/etc# grep forwarders /etc/alfis.conf | grep -v ^#
> forwarders = ["94.16.114.254:53", "94.247.43.254:53", "194.36.144.87:53", "1.1.1.1:53", "1.0.0.1:53"]

> root@R420:/etc# host linux-ru.lib 94.16.114.254:53
> host: couldn't get address for '94.16.114.254:53': not found
> root@R420:/etc# 
> root@R420:/etc# host linux-ru.lib 94.16.114.254
> Using domain server:
> Name: 94.16.114.254
> Address: 94.16.114.254#53
> Aliases: 
> 
> linux-ru.lib has address 90.156.230.27
> 

Why ?

**Comment by olej-tsil-51** [comment]
> No, You just have one forwarder, that doesn't know about those zones, and it is chosen by random.

Yes, it is so. If you remove the Cloudflare servers IP addresses from the array and leave only the OpenNIC DNS addresses, then EmerDNS name resolution occurs.

> olej@R420:~/2023/Yggdrasil/ygglkan.gpu$ grep forwarders /etc/alfis.conf | grep -v ^#
> forwarders = ["94.16.114.254:53", "94.247.43.254:53", "194.36.144.87:53"]

> olej@R420:~/2023/Yggdrasil$ host ygg.linux-ru.lib 127.0.0.4
> Using domain server:
> Name: 127.0.0.4
> Address: 127.0.0.4#53
> Aliases: 
> ygg.linux-ru.lib has IPv6 address 221:58c9:9a6:99be:f3d:c1ac:2b5b:9771

> olej@R420:~/2023/Yggdrasil$ host linux-ru.lib 127.0.0.4
> Using domain server:
> Name: 127.0.0.4
> Address: 127.0.0.4#53
> Aliases: 
> linux-ru.lib has address 90.156.230.27

This ticket can be considered completed!
